LibSWBF2/Wrappers/Terrain.cpp

```diff
@@ -187,93 +187,85 @@ namespace LibSWBF2::Wrappers
 	}
 
 
-	bool Terrain::GetIndexBuffer(ETopology requestedTopology, uint32_t& count, uint16_t*& indexBuffer) const
+	bool Terrain::GetIndexBuffer(ETopology requestedTopology, uint32_t& count, uint32_t*& indexBuffer) const
 	{
-		static List<uint16_t> indices;
-		static ETopology lastRequestedTopology;
```

Comment:
This was causing problems with repeated use over multiple lvl imports.  Indices is now a cached private member.  Index size also needed to be increased.